test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -55,21 +37,33 @@ describe('Request to go live: organisation name page', () => {
     }]
   }
 
-  const setupStubs = (serviceRole) => {
-    cy.task('setupStubs', simpleStub(serviceRole))
+  const serviceRoleBeforePatchRequests = utils.buildServiceRoleForMerchantDetailsField({ name: organisationName })
```

Comment:
Is this setup re-used enough to warrant going into the utils?
Also do we also want to explicitly set the go-live stage? I think the default for the fixture is 'NOT_STARTED' but we maybe shouldn't rely on that, and explicitly passing it in helps for visibility.

Reply:
Before we had this which is maybe clearer
```
const serviceRole = buildServiceRoleForGoLiveStage('NOT_STARTED')
serviceRole.service.merchant_details = { name: organisationName }
````

Reply:
It might be useful when we want to update other merchant details, which I think we still plan to do.

Reply:
I agree, it does look a lot clearer. I've updated this! 👍